python3.10 load.py --model-path /data/no-malicious/melo/TTS/myshell-ai-MeloTTS-English/checkpoint.pth --config-path /data/no-malicious/melo/TTS/myshell-ai-MeloTTS-English/config.json

melo library does not work on python 3.12.. switched to 3.10 worked. the official instruction specifies 3.9
